src/styles/blocks/card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  background: #fff;
+  border: 1px solid #e5e5e5;
+  border-radius: 5px;
+  max-width: $card-width;
+  transition: transform 300ms ease;
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__img {
+    margin: 32px auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    padding: 0 16px 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $color-main-accent;
+  }
+
+  &__id {
+    padding: 0 0 16px 16px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-secondary;
+  }
+
+  &__price {
+    display: flex;
+    height: 18px;
+    justify-content: space-between;
+    align-items: baseline;
+    padding: 0 16px;
+  }
+
+  &__text-title {
+    font-weight: 400;
+    font-size: 12px;
+    color: $color-secondary;
+  }
+
+  &__text-price {
+
+    font-weight: 700;
+    font-size: 16px;
+    color: $color-main-accent;
+  }
+
+  &__stars {
+    display: flex;
+    justify-content: space-between;
+    padding: 0 0 24px 16px ;
+    align-items: center;
+  }
+
+  &__reviews {
+    display: flex;
+    font-weight: 400;
+    font-size: 10px;
+    color: $color-main-accent;
+    padding-right: 16px;
+  }
+
+  &__button {
+    text-align: center;
+    text-decoration: none;
+    width: 166px;
+    height: 40px;
+    line-height: 40px;
+    background: $color-accent-blue;
+    border-radius: 5px;
+    font-weight: 700;
+    font-size: 14px;
+    text-transform: uppercase;
+    color: #fff;
+    margin: 16px;
+
+    &:hover {
+      background: #fff;
+      border: 1px solid $color-accent-blue;
```

Comment:
Better add a transparent border for the default state of the element and just change its color on hover.
Also, would be great to add transition for these properties too